CONTRIBUTING.md

```diff
@@ -22,55 +21,24 @@ Run `git config user.email` to see your Git email, and verify it with [your GitH
 
 ## Editor Config
 
-The [.editorconfig](https://github.com/webpack/webpack.js.org/blob/develop/.editorconfig) in the root should ensure consistent formatting. Please make sure you've [installed the plugin](http://editorconfig.org/#download) if your text editor needs one.
-
-## Git Flow
-
-We are using the [git flow](http://nvie.com/posts/a-successful-git-branching-model/) branching model. Please direct all pull requests to the [develop](https://github.com/webpack/webpack.js.org/tree/develop) branch.
+The [.editorconfig](https://github.com/webpack/webpack.js.org/blob/master/.editorconfig) in the root should ensure consistent formatting. Please make sure you've [installed the plugin](http://editorconfig.org/#download) if your text editor needs one.
 
 ## Branching Your Changes
 
-Making a branch in your fork for your contribution is helpful in the following ways:  
+Making a branch in your fork for your contribution is helpful in the following ways:
 
 * It allows you to have multiple contributions in as PRs at once.
 * It allows us to identify what your contribution is about from the branch name.
 
-You will want to checkout the `develop` branch locally before creating your new branch.
-
-There are two types of branches:
-
-* Feature
-* Hotfix
-
-### Features
-
-If your contribution is something new, like a new section or page or a new chunk to an existing page, you can create a branch with the following naming convention:  
-
-`feature/<the-new-feature>`
-
-So, for example, if you are adding a page on how to write a custom loader your branch could have the name:
-
-`feature/add-custom-loaders-page`
-
-### Fixes
-
-If you are fixing existing content or code you can create a branch with the following naming convention:
-
-`hotfix/<the-fix>`
-
-So, for example, if you are fixing spelling on the api configuration page for an option, your branch could have the name:
-
-`hotfix/fix-option-property-spelling-on-api-configuration`
-
 ## Submitting Changes
 
 After getting some feedback, push to your fork branch and submit a pull request. We may suggest some changes or improvements or alternatives, but for small changes your pull request should be accepted quickly.
```

Comment:
We could somewhere link to https://help.github.com/articles/proposing-changes-to-your-work-with-pull-requests/